src/Components/Carousel/Carousel.js

```diff
@@ -36,15 +35,15 @@ const Carousel = () => {
     };
   }, []);
 
-  useEffect(() => {
-    if (offset === 0) {
-      setActivePage(0);
-    } else if (offset > 0 && offset < numberReq - count) {
-      setActivePage(1);
-    } else if (offset + count == numberReq) {
-      setActivePage(2);
-    }
-  }, [offset, numberReq, count]);
+  // useEffect(() => {
+  //   if (offset === 0) {
+  //     setActivePage(0);
+  //   } else if (offset > 0 && offset < numberReq - count) {
+  //     setActivePage(1);
+  //   } else if (offset + count == numberReq) {
+  //     setActivePage(2);
+  //   }
+  // }, [offset, numberReq, count]);
```

Comment:
If commented code is not needed, it's better to remove it

Reply:
done